### ISSUE TITLE
[feature request] Autosync

### ISSUE THREAD

**Issue by kacara** [flackbash/AudioAnchor#133]
For short track listeners, it would be useful to automatically sync new tracks each time the app is opened. Maybe an option of this kind would be useful. 
I can not think of any harm of setting it even as a default behaviour w/o an option. Maybe takes long time for big databases?

**Comment by flackbash** [comment]
Yes, exactly, for big databases it takes long, so I can't set it as default behavior. 
But I agree that an option like this could be useful to some users.
Are you aware of the "swipe down" method for syncing? I.e. you don't have to go to the menu first and click on Synchronize.
